pkg/exec/action.go

```diff
@@ -38,6 +38,7 @@ func (a Action) GetSteps() []builder.ExecutableStep {
 }
 
 var _ builder.ExecutableStep = Step{}
+var _ builder.HasCustomDashes = Step{}
```

Comment:
I think that it would still work with this (line 41) and lines 68 -71 removed. The interface for HasCustomDashes is optional and since you are doing the default, there's no need to implement it, right?

Reply:
Oh yes!  Thank you, good call; removed.